vendor/genfs_contexts

```diff
@@ -1,4 +1,14 @@
 genfscon proc /irq                                    u:object_r:proc_irq:s0
+genfscon proc /sys/vm/swappiness                      u:object_r:proc_swappiness:s0
+genfscon proc /sys/kernel/sched_boost                 u:object_r:proc_sched:s0
```

Comment:
The `sched_*` labels could be moved to the platforms, depending on which features they have.

Reply:
@alviteri hat do you think?